README.md

```diff
@@ -10,10 +10,6 @@ appropriately for your cluster in the `default` namespace:
 
 ## Prerequisites
 
-### Knative Serving
-
-The `Service` CRD from [Knative Serving](https://github.com/knative/serving) is required.
```

Comment:
I think for the `GH` source, the `controller-manager` (from `eventing-sources` repo) needs it. 

In the long run, as discussed yesterday on slack. This ONLY installs the "core" (the `knative/eventing` repo). and for that project, we can remove it.   

I tend to keep it for the 0.6.0 release